src/tools/vfs/vfs_server.c

```diff
@@ -32,6 +32,7 @@ typedef struct {
 } vfs_serever_fd_state_t;
 
 typedef struct {
+    struct sigaction       old_sigact;
```

Comment:
why do we care about prev sighandler?

Reply:
otherwise, IO-demo signal handler isn't called

Reply:
do we have another signal handler?

Reply:
btw, it also could be problematic for user's application which set the same signals which are handled by VFS.

Reply:
vfs_server is an application , not part of libucs
it should not affect io_demo
also io_demo is an application so no other application could exist when it's running

Reply:
I thought that VFS is a part of UCX library.
Reverted code that takes care of old sigactions.

Reply:
> I thought that VFS is a part of UCX library.
> Reverted code that takes care of old sigactions.

Some VFS common infra is part of libucs, but the daemon (ucx_vfs mount tool) is standalone, like ucx_perftest